guides/doc-Installing_Proxy/master.adoc

```diff
@@ -23,7 +23,17 @@ include::common/assembly_performing-additional-configuration-on-capsule-server.a
 // Configuring {SmartProxyServer} with External Services
 include::common/assembly_configuring-external-services.adoc[leveloffset=+1]
 
+include::common/assembly_managing-dhcp-on-smart-proxies.adoc[leveloffset=+1]
+
+include::common/assembly_managing-dns-on-smart-proxies.adoc[leveloffset=+1]
```

Comment:
Technically these options all also apply to the `doc-Installing_Server` side. I'd be OK with doing that in a follow up, but it's something to think about.

Reply:
👍